### PR TITLE
Update sidenav render children

### PR DESCRIPTION
**Pull request by jlhogan**

### CODE REVIEW

**Review by bamader — APPROVED** [review]
LGTM

File: src/components/SideNav/SideNav.tsx [hunk 1]
```diff
@@ -55,9 +47,8 @@ export const SideNav = ({
           { 'mobile-lg:grid-col-12': items.length === 0 }
         )}
       >
-        {activeItemIndex !== undefined && activeItemIndex < items.length
-          ? items[activeItemIndex].content
-          : noActiveItemContent}
+        {children ||
```
Comment (mmazanec22) [inline src/components/SideNav/SideNav.tsx:50]: Yay it makes me happy that this is similar to how Tab Nav works, feels like we're getting toward consistency

File: src/components/SideNav/SideNav.tsx [hunk 2]
```diff
@@ -4,44 +4,36 @@ import { SideNavItem, SideNavItemProps } from './SideNavItem';
...
 
   useEffect(() => {
-    setActiveItemIndex(
-      // eslint-disable-next-line eqeqeq
-      externalActiveItemIndex != undefined && !isNaN(externalActiveItemIndex)
-        ? externalActiveItemIndex
-        : undefined
-    );
-  }, [externalActiveItemIndex]);
+    setActiveId(activeItemId);
```
Comment (mmazanec22) [inline src/components/SideNav/SideNav.tsx:18]: Hooray for this simplification!

**Review by mmazanec22 — APPROVED** [review]

File: src/components/SideNav/SideNav.tsx (hunk 1 above)
Comment (jlhogan) [reply src/components/SideNav/SideNav.tsx:50]: all thanks to u ! i wrote sidenav a while ago when i knew much less about how all this actually works :)

**Pull request merged**